yt/go/yt/integration/cypress_client_test.go

```diff
@@ -261,6 +263,28 @@ func (s *Suite) TestLinkNode(t *testing.T, yc yt.Client) {
 	require.Equal(t, yt.NodeLink, typ)
 }
 
+func (s *Suite) TestConcatNode(t *testing.T, yc yt.Client) {
+	t.Parallel()
+	var err error
+
+	ctx, cancel := context.WithTimeout(s.Ctx, time.Second*30)
+	defer cancel()
+
+	src1 := tmpPath()
+	src2 := tmpPath()
+	dst := tmpPath()
+
+	_, err = yc.CreateNode(ctx, dst, yt.NodeTable, nil)
+	require.NoError(t, err)
+	_, err = yc.CreateNode(ctx, src1, yt.NodeTable, nil)
+	require.NoError(t, err)
+	_, err = yc.CreateNode(ctx, src2, yt.NodeTable, nil)
+	require.NoError(t, err)
+
+	err = yc.ConcatNode(ctx, []ypath.YPath{src1, src2}, dst, nil)
```

Comment:
I think it is best to keep the name consistent across different sdks, it is called Concatenate in C++ and yt concatenate in cli/python, so probably `yc.Concatenate` is better

Reply:
fixed method name